sdk/eventhubs/azure-messaging-eventhubs/pom.xml

```diff
@@ -37,7 +37,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.6.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.7.0-beta.3</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
```

Comment:
@jianghaolu Could you add a note here why were are doing this?

Reply:
Actually this is not needed. reverted this change.